virttest/qemu_vm.py

```diff
@@ -1119,8 +1119,8 @@ def set_value(opt_string, key, fallback=None):
             set_value("agent-mouse=%s", "spice_agent_mouse")
             set_value("playback-compression=%s", "spice_playback_compression")
 
-            set_yes_no_value("spice_ipv4", yes_value="ipv4")
-            set_yes_no_value("spice_ipv6", yes_value="ipv6")
+            set_yes_no_value("spice_ipv4", yes_value="ipv4=on")
+            set_yes_no_value("spice_ipv6", yes_value="ipv6=on")
```

Comment:
Before:
`spice_ipv4=yes`  ==> `ipv4` in command line, and `spice_ipv4=no` mean no ipv4. 
And now:
`spice_ipv4=yes`  ==> `ipv4=on` in command line, and `spice_ipv4=on` can not work with the current design, no way to add `ipv4=off` in command line although it is not very meaningful.

I think it should support add `option=off` instead of remove the option. how do you think ?

Reply:
@yanan-fu It is supported actually, set_yes_no_value("spice_ipv4", no_value="ipv4=off") could get the desired command line, but originally the code has no this intention neither, I think not necessary to introduce it, but here should ask about the feature owner's opinion, and I heard that spice is deprecated too, not sure, @zhguo01, would you help check here? Thanks.

Reply:
You are right, `option=off` can work.  Base on this, i do not have strong idea about change `yes/no` to `on/off`.

And yes, spice will be deprecated, so i am ok with that.